android/src/main/java/com/example/flutter_segment/FlutterSegmentPlugin.java

```diff
@@ -54,18 +55,6 @@ private void setupChannels(Context applicationContext, BinaryMessenger messenger
     methodChannel = new MethodChannel(messenger, "flutter_segment");
     // register the channel to receive calls
     methodChannel.setMethodCallHandler(this);
-
-    try {
-      ApplicationInfo ai = applicationContext.getPackageManager()
-              .getApplicationInfo(applicationContext.getPackageName(), PackageManager.GET_META_DATA);
-
-      Bundle bundle = ai.metaData;
-
-      FlutterSegmentOptions options = FlutterSegmentOptions.create(bundle);
-      setupChannels(options);
-    } catch (Exception e) {
-      Log.e("FlutterSegment", e.getMessage());
-    }
```

Comment:
Whats the idea behind this change?

Reply:
We don't need this code anymore since these are deprecated:
<img width="988" alt="image" src="https://user-images.githubusercontent.com/24664153/158910296-f1af31af-f77b-4e43-abcd-11130c1df985.png">

and also in iOS side.

Reply:
Got it, then it does not relate to this PR right?

Reply:
@danielgomezrico I think it's related because it is a breaking change since we can't use an array in the android manifest or .plist. So, We need to remove it then.

Reply:
1. The old implementation (Integration using AndroidManifest.xml and ios info.plist) is deprecated.
2. what I propose here is a new method that will not be compatible with the old implementation. Breaking change.

@danielgomezrico ,

Reply:
> 1. The old implementation (Integration using AndroidManifest.xml and ios info.plist) is deprecated.
> 
>     2. what I propose here is a new method that will not be compatible with the old implementation. Breaking change.
> 
> 
> @danielgomezrico ,

@ariefwijaya there is another issue open where initializing the sdk using the dart initialization results in App Installed events not being tracked on iOS:
https://github.com/la-haus/flutter-segment/issues/26

So while the old implementation is marked as deprecated, we had to revert back to it and rely on it for our iOS integrations

Reply:
Oh Thanks for the info, but in my case, I don't get that problem and not related to my implementation.
Because, I just improve our existing dart only installation.. 
You can continue the discussion about Installed Events not being tracked in this issue #26